src/main/java/org/jenkinsci/plugins/github/util/FluentIterableWrapper.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (C) 2008 The Guava Authors
```

Comment:
This is a temporary solution that will be removed when jenkins will have newer dependencies or java8.